src/persist/benches/writer.rs

```diff
@@ -66,40 +119,73 @@ fn block_on<T, F: FnOnce(FutureHandle<T>)>(f: F) -> Result<T, Error> {
     rx.recv()
 }
 
+// Benchmark the write throughput of Indexed::write.
 fn bench_write<L: Log, B: Blob>(
     index: &mut Indexed<L, B>,
     id: Id,
     updates: Vec<((Vec<u8>, Vec<u8>), u64, isize)>,
     b: &mut Bencher,
 ) {
-    b.iter(move || {
-        // We intentionally never call seal, so that the data only gets written
-        // once to Unsealed, and not to Trace.
-        block_on_drain(index, |i, handle| {
-            i.write(vec![(id, updates.clone())], handle)
-        })
-        .unwrap();
+    b.iter_custom(|iters| {
```

Comment:
document that the intention here is to avoid benchmarking the clone

Reply:
nit: iter_batched might be exactly what you want here? even if so, i don't care if you switch because this is perfectly readable to me